### DISCUSSION TITLE
Upgrade process using docker ❔

### DISCUSSION THREAD

**Discussion by lfr** [datalust/seq-tickets#1317] [Help]
The "Upgrade from earlier versions" documentation entry only mentions the windows server upgrade process. What's the upgrade process when Seq is installed using docker?

**Comment by nblumhardt** [comment]
Hi Luis - thanks for pointing out the shortcomings of the docs, here; we'll try to get them updated for the next release 👍 

The process when using Docker is generally just to restart the container running the new image version.

If you're running Docker directly, the usual way to do this is to remove the current container and launch a new one with the same data volume mounted. (If you ended up storing data directly on the container's filesystem, instead of in a host volume mount, there's some discussion in [this answer](https://stackoverflow.com/a/33536436/138206) that might help.)

If you're using Swarm, K8s/Helm, or another orchestration platform, you'll need to follow the upgrade process for that platform. Let me know if you need help with any of these.

The compatibility notes/breaking changes listed in the existing docs are still generally valid.

**Reply by mpp-olano** [reply]
Thanks
